src/extractHangul.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { extractHangul } from './extractHangul';
+
+describe('extractHangul 함수 테스트', () => {
```

Comment:
일관성을 유지하기 위해서 , 함수 테스트 글자를 지우는것은 어떨까요?

```suggestion
describe('extractHangul', () => {
```

Reply:
넵 !